jinja2/utils.py

```diff
@@ -39,6 +39,15 @@
 
 concat = u''.join
 
+def custom_concat(to_concat):
+    def format_args(arg):
+        if "'" in arg:
```

Comment:
this doesn't break anything?! it looks like the kind of code that would easily break things

Reply:
This is the only way i have found to  parse 'format' list in visit_Output function